modules/deployment-plug-in-cluster.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+
+[id="build-image-with-docker"]
+= Build an image with Docker
+In order to deploy your plug-in on a cluster, you need to build an image and push it to an image directory.
+
+.Procedure
+
+. Build the image with the following command:
++
+[source,terminal]
+----
+$ docker build -t quay.io/my-repositroy/my-plugin:latest
```

Comment:
We should find out if we give examples using `docker` or `podman` elsewhere in the docs. (Either will work with the same arguments.)